### PR TITLE
Increased legibility of README.

### PR DESCRIPTION
**Pull request by kirstywilliams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,38 +2,28 @@
 
 ## Overview
 
-DistributedLog (DL) is a high-performance, replicated log service, offering durability, replication and strong consistency
-as essentials for building reliable distributed systems.
+DistributedLog (DL) is a high-performance, replicated log service, offering durability, replication and high consistency as essentials for building reliable distributed systems.
```
Comment (sijie) [inline README.md]: keep 'strong consistency'?

**Comment by kirstywilliams** [comment]
Either works :). Updated.

**Comment by leighst** [comment]
There was an effort in the original text to keep lines short - idea is to make sure its readable even if one is looking at the source.

Can we keep the grammar changes (which are great!) and leave the line breaks?

**Comment by kirstywilliams** [comment]
Max char count of 80 per line added.

**Comment by sijie** [comment]
@kirstywilliams could you rebase your change? so I could merge.

**Comment by kirstywilliams** [comment]
@sijie: done!

**Comment by sijie** [comment]
LGTM +1

**Pull request merged**